samples/builder/src/main/module.gwt.xml

```diff
@@ -29,4 +29,5 @@
     <inherits name="org.jboss.gwt.elemento.sample.Common"/>
 
     <entry-point class="org.jboss.gwt.elemento.sample.builder.client.Main"/>
+    <collapse-all-properties/>
```

Comment:
this speed up a bit samples compilation... and the final js is almost the same size, but instead of 5 copies just one, so the CI/local-target use much less space.